drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -283,6 +283,7 @@ function DragAndDropBlock(runtime, element, configuration) {
     if (!window.gettext) { window.gettext = function gettext_stub(string) { return string; }; }
 
     var $element = $(element);
+    element = $element[0]; // <- workaround Studio bug until https://github.com/edx/edx-platform/pull/11433 is merged
```

Comment:
@dianakhuang Do you think it's ok if we leave it in for a while though, in order to preserve Dogwood compatibility? We have clients who are using this block on Dogwood.